core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -1141,7 +1139,10 @@ class FsHistoryProviderSuite extends SparkFunSuite with Matchers with Logging {
     assert(mockedProvider.isBlacklisted(accessDeniedPath))
     clock.advance(24 * 60 * 60 * 1000 + 1) // add a bit more than 1d
     mockedProvider.cleanLogs()
-    assert(!mockedProvider.isBlacklisted(accessDeniedPath))
+    updateAndCheck(mockedProvider) { list =>
+      assert(!mockedProvider.isBlacklisted(accessDeniedPath))
+      list.size should be(2)
```

Comment:
Previously, this always 1 because the newly released `accessDeniedPath` is ignored by `shouldReloadLog` due to the unchanged size.

Reply:
shall we add a comment in order to explain this? Can we also check that accessDeniedPath is present in the list? Thanks!

Reply:
I updated like the following. So, it explains itself without comments.
```
+      list.exists(_.name == "accessDenied")
+      list.exists(_.name == "accessGranted")
+      list.size should be(2)
```